### DISCUSSION TITLE
Integration with FastAPI

### DISCUSSION THREAD

**Discussion by pchanial** [wyfo/apischema#126] [General]
Hello, have you looked at how easy or difficult it would be use apischema in FastAPI (for automatic deserialization of arguments, serialization of responses, and OpenAPI spec generation) instead of Pydantic? Either as a FastAPI monkey-patching extension or proposing them to have serialization/deserialization/validation interfaces so that other packages could be used. I don't know how deep are the ties between FastAPI and Pydantic.

**Comment by c00kiemon5ter** [comment]
I'd be interested in this too :)

**Reply by coretl** [reply]
Looks like other people are interested too... https://github.com/tiangolo/fastapi/issues/3365
